ldap-auth/kafka/consumer.properties

```diff
@@ -0,0 +1,6 @@
+sasl.mechanism=SCRAM-SHA-256
+security.protocol=SASL_PLAINTEXT
+sasl.jaas.config=org.apache.kafka.common.security.scram.ScramLoginModule required \
```

Comment:
This look invalid @sknop , it should be PlainLoginModule no?

Reply:
Good spot, the whole file has to go - there is no consumer user anymore. Fixed